Library/Formula/cim.rb

```diff
@@ -0,0 +1,659 @@
+class Cim < Formula
+  desc "GNU Simula"
+  homepage "https://www.gnu.org/software/cim/"
+  url "http://ftpmirror.gnu.org/cim/cim-5.1.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/cim/cim-5.1.tar.gz"
+  sha256 "b90717b66ec400503bdd69e537e8b7f3e8a9d106c3ba9a08a04ae57369a069a9"
+
+  # patch submitted upstream, no response yet.
```

Comment:
Was it submitted via email, or to a bug tracker? If the former, add the submission date; if the latter, add the ticket URL in the comment, thanks!